.github/workflows/azure-integration.yml

```diff
@@ -0,0 +1,88 @@
+name: Azure BlobStore Integration
+
+on:
+  push:
+    branches:
+      - "**"
+    paths:
+      - ".github/workflows/azure-integration.yml"
+      - "pom.xml"
+      - "geowebcache/pom.xml"
+      - "geowebcache/core/**"
+      - "geowebcache/azureblob/**"
+  pull_request:
+    branches:
+      - "main"
+    paths:
+      - ".github/workflows/azure-integration.yml"
+      - "pom.xml"
+      - "geowebcache/pom.xml"
+      - "geowebcache/core/**"
+      - "geowebcache/azureblob/**"
```

Comment:
Github actions in GT/GWC/GS normally only have:

```
on: [pull_request]
```

Builds are not run on pushes (too late), and are run on all pull request regardless of the branch.

Reply:
nice, `branches:*` was a leftover from working on my fork to be able of run them before the pr.
Changed it to run an all pr's regardless of the branch, and on all tags.